### DISCUSSION TITLE
WAV file read: int, long, or double

### DISCUSSION THREAD

**Discussion by LiveNathan** [psambit9791/jDSP#54] [Q&A] [answered]
Hi! Why do I need to choose this? How do I make the decision?

Here's an example file I'm reading: https://s3.amazonaws.com/appforest_uf/f1660040393454x716545782075606000/Y12%20ARC-48k.wav

**Comment by psambit9791** [comment, marked as answer]
This just selects the datatype that you want to load the data as.

**Reply by LiveNathan** [reply]
Ah, ok. I'm not super familiar with WAV format, but isn't it natively int values?
Ultimately, I'm going to convert it to double to perform a normalization, so I guess I might as well read it in as doubles.

**Comment by psambit9791** [comment]
They are but in case the user needs scaled values as doubles, we can use
double to read the file as a 2D array of double size.
